Svelte errors displayed during the preprocessing step while running the UI container

## Description
When I start the UI container using the `docker exec -it mathesar_ui_1 bash` command, Svelte compiler throws some errors.

## Expected behavior
I expected the docker container to run without any issues

## Environment
 - Docker Desktop Version: 3.5.2.18
 - Docker CLI Version: 20.10.7, build f0df350

## Additional context
```
> mathesar_ui@0.0.0 preinstall /home/node/ui
> npx npm-force-resolutions

audited 2372 packages in 16.436s

247 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities


> mathesar_ui@0.0.0 dev /home/node/ui
> vite


npx: installed 6 in 2.893s
npm WARN eslint-config-airbnb@18.2.1 requires a peer of eslint-plugin-jsx-a11y@^6.4.1 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-config-airbnb@18.2.1 requires a peer of eslint-plugin-react@^7.21.5 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-config-airbnb@18.2.1 requires a peer of eslint-plugin-react-hooks@^4 || ^3 || ^2.3.0 || ^1.7.0 but none is installed. You must install peer dependencies yourself.
npm WARN mathesar_ui@0.0.0 No repository field.
npm WARN mathesar_ui@0.0.0 No license field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"arm64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/watchpack-chokidar2/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"arm64"})

 > html:/home/node/ui/src/sections/table-view/status-pane/StatusPane.svelte:2:23: error: Expected ";" but found ":"
    2 │   export let totalCount: number;
      ╵                        ^

 > html:/home/node/ui/src/sections/table-view/actions-pane/ActionsPane.svelte:3:14: error: Expected "from" but found "{"
    3 │   import type {
      ╵               ^

error when starting dev server:
Error: Build failed with 2 errors:
html:/home/node/ui/src/sections/table-view/actions-pane/ActionsPane.svelte:3:14: error: Expected "from" but found "{"
html:/home/node/ui/src/sections/table-view/status-pane/StatusPane.svelte:2:23: error: Expected ";" but found ":"
    at failureErrorWithLog (/home/node/ui/node_modules/esbuild/lib/main.js:1449:15)
    at /home/node/ui/node_modules/esbuild/lib/main.js:1131:28
    at runOnEndCallbacks (/home/node/ui/node_modules/esbuild/lib/main.js:921:63)
    at buildResponseToResult (/home/node/ui/node_modules/esbuild/lib/main.js:1129:7)
    at /home/node/ui/node_modules/esbuild/lib/main.js:1236:14
    at /home/node/ui/node_modules/esbuild/lib/main.js:609:9
    at handleIncomingPacket (/home/node/ui/node_modules/esbuild/lib/main.js:706:9)
    at Socket.readFromStdout (/home/node/ui/node_modules/esbuild/lib/main.js:576:7)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:290:12)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! mathesar_ui@0.0.0 dev: `vite`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the mathesar_ui@0.0.0 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/node/.npm/_logs/2021-07-31T09_40_31_375Z-debug.log
npx: installed 6 in 0.91s
npm WARN eslint-config-airbnb@18.2.1 requires a peer of eslint-plugin-jsx-a11y@^6.4.1 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-config-airbnb@18.2.1 requires a peer of eslint-plugin-react@^7.21.5 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-config-airbnb@18.2.1 requires a peer of eslint-plugin-react-hooks@^4 || ^3 || ^2.3.0 || ^1.7.0 but none is installed. You must install peer dependencies yourself.
npm WARN mathesar_ui@0.0.0 No repository field.
npm WARN mathesar_ui@0.0.0 No license field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"arm64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/watchpack-chokidar2/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"arm64"})

 > html:/home/node/ui/src/sections/table-view/actions-pane/ActionsPane.svelte:3:14: error: Expected "from" but found "{"
    3 │   import type {
      ╵               ^

 > html:/home/node/ui/src/sections/table-view/status-pane/StatusPane.svelte:2:23: error: Expected ";" but found ":"
    2 │   export let totalCount: number;
      ╵                        ^

error when starting dev server:
Error: Build failed with 2 errors:
html:/home/node/ui/src/sections/table-view/actions-pane/ActionsPane.svelte:3:14: error: Expected "from" but found "{"
html:/home/node/ui/src/sections/table-view/status-pane/StatusPane.svelte:2:23: error: Expected ";" but found ":"
    at failureErrorWithLog (/home/node/ui/node_modules/esbuild/lib/main.js:1449:15)
    at /home/node/ui/node_modules/esbuild/lib/main.js:1131:28
    at runOnEndCallbacks (/home/node/ui/node_modules/esbuild/lib/main.js:921:63)
    at buildResponseToResult (/home/node/ui/node_modules/esbuild/lib/main.js:1129:7)
    at /home/node/ui/node_modules/esbuild/lib/main.js:1236:14
    at /home/node/ui/node_modules/esbuild/lib/main.js:609:9
    at handleIncomingPacket (/home/node/ui/node_modules/esbuild/lib/main.js:706:9)
    at Socket.readFromStdout (/home/node/ui/node_modules/esbuild/lib/main.js:576:7)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:290:12)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! mathesar_ui@0.0.0 dev: `vite`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the mathesar_ui@0.0.0 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/node/.npm/_logs/2021-07-31T09_54_56_610Z-debug.log
npx: installed 6 in 3.486s
npm WARN eslint-config-airbnb@18.2.1 requires a peer of eslint-plugin-jsx-a11y@^6.4.1 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-config-airbnb@18.2.1 requires a peer of eslint-plugin-react@^7.21.5 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-config-airbnb@18.2.1 requires a peer of eslint-plugin-react-hooks@^4 || ^3 || ^2.3.0 || ^1.7.0 but none is installed. You must install peer dependencies yourself.
npm WARN mathesar_ui@0.0.0 No repository field.
npm WARN mathesar_ui@0.0.0 No license field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"arm64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/watchpack-chokidar2/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"arm64"})

 > html:/home/node/ui/src/sections/table-view/status-pane/StatusPane.svelte:2:23: error: Expected ";" but found ":"
    2 │   export let totalCount: number;
      ╵                        ^

 > html:/home/node/ui/src/sections/table-view/actions-pane/ActionsPane.svelte:3:14: error: Expected "from" but found "{"
    3 │   import type {
      ╵               ^

error when starting dev server:
Error: Build failed with 2 errors:
html:/home/node/ui/src/sections/table-view/actions-pane/ActionsPane.svelte:3:14: error: Expected "from" but found "{"
html:/home/node/ui/src/sections/table-view/status-pane/StatusPane.svelte:2:23: error: Expected ";" but found ":"
    at failureErrorWithLog (/home/node/ui/node_modules/esbuild/lib/main.js:1449:15)
    at /home/node/ui/node_modules/esbuild/lib/main.js:1131:28
    at runOnEndCallbacks (/home/node/ui/node_modules/esbuild/lib/main.js:921:63)
    at buildResponseToResult (/home/node/ui/node_modules/esbuild/lib/main.js:1129:7)
    at /home/node/ui/node_modules/esbuild/lib/main.js:1236:14
    at /home/node/ui/node_modules/esbuild/lib/main.js:609:9
    at handleIncomingPacket (/home/node/ui/node_modules/esbuild/lib/main.js:706:9)


> mathesar_ui@0.0.0 preinstall /home/node/ui
> npx npm-force-resolutions

audited 2372 packages in 14.071s

247 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities


> mathesar_ui@0.0.0 dev /home/node/ui
> vite


> mathesar_ui@0.0.0 preinstall /home/node/ui
> npx npm-force-resolutions

audited 2372 packages in 17.304s

247 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities


> mathesar_ui@0.0.0 dev /home/node/ui
> vite


> mathesar_ui@0.0.0 preinstall /home/node/ui
> npx npm-force-resolutions

audited 2372 packages in 14.293s

247 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities


> mathesar_ui@0.0.0 dev /home/node/ui
> vite


> mathesar_ui@0.0.0 preinstall /home/node/ui
> npx npm-force-resolutions

audited 2372 packages in 14.023s

247 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities


> mathesar_ui@0.0.0 dev /home/node/ui
> vite


    at Socket.readFromStdout (/home/node/ui/node_modules/esbuild/lib/main.js:576:7)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:290:12)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! mathesar_ui@0.0.0 dev: `vite`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the mathesar_ui@0.0.0 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/node/.npm/_logs/2021-07-31T09_57_54_997Z-debug.log
npx: installed 6 in 0.961s
npm WARN eslint-config-airbnb@18.2.1 requires a peer of eslint-plugin-jsx-a11y@^6.4.1 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-config-airbnb@18.2.1 requires a peer of eslint-plugin-react@^7.21.5 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-config-airbnb@18.2.1 requires a peer of eslint-plugin-react-hooks@^4 || ^3 || ^2.3.0 || ^1.7.0 but none is installed. You must install peer dependencies yourself.
npm WARN mathesar_ui@0.0.0 No repository field.
npm WARN mathesar_ui@0.0.0 No license field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"arm64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/watchpack-chokidar2/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"arm64"})

 > html:/home/node/ui/src/sections/table-view/actions-pane/ActionsPane.svelte:3:14: error: Expected "from" but found "{"
    3 │   import type {
      ╵               ^

 > html:/home/node/ui/src/sections/table-view/status-pane/StatusPane.svelte:2:23: error: Expected ";" but found ":"
    2 │   export let totalCount: number;
      ╵                        ^

error when starting dev server:
Error: Build failed with 2 errors:
html:/home/node/ui/src/sections/table-view/actions-pane/ActionsPane.svelte:3:14: error: Expected "from" but found "{"
html:/home/node/ui/src/sections/table-view/status-pane/StatusPane.svelte:2:23: error: Expected ";" but found ":"
    at failureErrorWithLog (/home/node/ui/node_modules/esbuild/lib/main.js:1449:15)
    at /home/node/ui/node_modules/esbuild/lib/main.js:1131:28
    at runOnEndCallbacks (/home/node/ui/node_modules/esbuild/lib/main.js:921:63)
    at buildResponseToResult (/home/node/ui/node_modules/esbuild/lib/main.js:1129:7)
    at /home/node/ui/node_modules/esbuild/lib/main.js:1236:14
    at /home/node/ui/node_modules/esbuild/lib/main.js:609:9
    at handleIncomingPacket (/home/node/ui/node_modules/esbuild/lib/main.js:706:9)
    at Socket.readFromStdout (/home/node/ui/node_modules/esbuild/lib/main.js:576:7)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:290:12)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! mathesar_ui@0.0.0 dev: `vite`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the mathesar_ui@0.0.0 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/node/.npm/_logs/2021-07-31T09_58_14_561Z-debug.log
npx: installed 6 in 0.829s
npm WARN eslint-config-airbnb@18.2.1 requires a peer of eslint-plugin-jsx-a11y@^6.4.1 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-config-airbnb@18.2.1 requires a peer of eslint-plugin-react@^7.21.5 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-config-airbnb@18.2.1 requires a peer of eslint-plugin-react-hooks@^4 || ^3 || ^2.3.0 || ^1.7.0 but none is installed. You must install peer dependencies yourself.
npm WARN mathesar_ui@0.0.0 No repository field.
npm WARN mathesar_ui@0.0.0 No license field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"arm64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/watchpack-chokidar2/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"arm64"})

 > html:/home/node/ui/src/sections/table-view/status-pane/StatusPane.svelte:2:23: error: Expected ";" but found ":"
    2 │   export let totalCount: number;
      ╵                        ^

 > html:/home/node/ui/src/sections/table-view/actions-pane/ActionsPane.svelte:3:14: error: Expected "from" but found "{"
    3 │   import type {
      ╵               ^

error when starting dev server:
Error: Build failed with 2 errors:
html:/home/node/ui/src/sections/table-view/actions-pane/ActionsPane.svelte:3:14: error: Expected "from" but found "{"
html:/home/node/ui/src/sections/table-view/status-pane/StatusPane.svelte:2:23: error: Expected ";" but found ":"
    at failureErrorWithLog (/home/node/ui/node_modules/esbuild/lib/main.js:1449:15)
    at /home/node/ui/node_modules/esbuild/lib/main.js:1131:28
    at runOnEndCallbacks (/home/node/ui/node_modules/esbuild/lib/main.js:921:63)
    at buildResponseToResult (/home/node/ui/node_modules/esbuild/lib/main.js:1129:7)
    at /home/node/ui/node_modules/esbuild/lib/main.js:1236:14
    at /home/node/ui/node_modules/esbuild/lib/main.js:609:9
    at handleIncomingPacket (/home/node/ui/node_modules/esbuild/lib/main.js:706:9)
    at Socket.readFromStdout (/home/node/ui/node_modules/esbuild/lib/main.js:576:7)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:290:12)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! mathesar_ui@0.0.0 dev: `vite`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the mathesar_ui@0.0.0 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/node/.npm/_logs/2021-07-31T09_59_33_556Z-debug.log
```

--- a/mathesar_ui/src/sections/table-view/actions-pane/ActionsPane.svelte
+++ b/mathesar_ui/src/sections/table-view/actions-pane/ActionsPane.svelte
@@ -1,4 +1,4 @@
-<script type="ts">
+<script lang="ts">
   import { createEventDispatcher } from 'svelte';
   import type {
     TableColumnStore,

--- a/mathesar_ui/src/sections/table-view/status-pane/StatusPane.svelte
+++ b/mathesar_ui/src/sections/table-view/status-pane/StatusPane.svelte
@@ -1,4 +1,4 @@
-<script type="ts">
+<script lang="ts">
   export let totalCount: number;
   export let selectedEntries: string[];
 </script>

